antidox/pii_results.txt

```diff
@@ -0,0 +1,16 @@
+My phone number is (206) 555-0123.
```

Comment:
I think you might want to add both this file and the other toxicity_results.txt to `.gitignore`.  These are just the results for our test data, but they don't need to be added to GitHub

Reply:
It is fairly common to check in testdata files in a testdata directory, although personally I prefer tests that do everything as code.

Reply:
Just discussed with Jeff offline, let's add this and toxicity_results.txt to .gitignore, but for future data used on by tests we can put those in a testdata directory